translations/construction-project/de.yaml

```diff
@@ -63,6 +63,7 @@ ember-gwr:
             countryNameShortPlaceholder: Wählen Sie ein Land...
 
       typeOfConstructionProject: Bauprojekttyp
+      typeOfConstructionProjectHint: Für Tiefbauprojekte sind keine Gebäude zugelassen.
```

Comment:
Isnt this string already defined somwhere?

Reply:
Yes, I replaced it :+1: